index.html

```diff
@@ -4,9 +4,136 @@
+      <li id="aboutmenu"  class="links"><a href="about.html">About</a></li>
+      <li class="links"><a href="#">Program</a></li>
+      <li class="links"><a href="#">Sponsor</a></li>
+      <li class="links"><a href="#">News</a></li>
+    </nav>
+    <button type="button" class="linkss">CC Campaign</button>
+  </header>
+  <section >
+    <div class="introduction"> 
+    <h1 id="hello-text">"Hello sharing world!"</h1>
+    <h2 id="summer-text">CREATIVE SUMMER CONCERT SERIES 2023</h2>
+    <div id="details"><P class="primary-text">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.</P></div>
+      <div id="location">
+        <h3 id="date">2023.10.18(THUR) ~ 22TH(MON)</h3>
+        <p id="place">@ National museum of Korea, Art center Nabi and more</p>
+      </div>
+    </div>
+  </section>
+  <section class="main-program">
+    <h2 id="program">Main Program</h2>
+    <br>
+    <img src="./images/title_underline.png" alt="" class="underlineimg"/>
+    <br><br>
+    <div id="program-list">
+    <div id="one">
+      <ul class="list">
+        <li class="main"><img src="./images/conference.png" alt="" class="program-img"></li>
+        <li class="main"><h4>Lecture</h4></li>
+        <li class="main list-para"><p>Listen to the speakers from various countries about the messages of sharing and opening.</p></li>
+      </ul>
+    </div>
+    <div id="two">
+      <ul class="list">
+        <li class="main"><img src="./images/exhibition.png" alt="" class="program-img"></li>
+        <li class="main"><h4>CC Exhibition</h4></li>
+        <li class="main list-para"><p>Appreciate various creations applying CC license of artists, organized from Art Center Nabi</p></li>
+      </ul>
+    </div>
+    <div id="three">
+      <ul class="list">
+        <li class="main"><img src="./images/forum.png" alt="" class="program-img"></li>
+        <li class="main"><h4>Forum</h4></li>
+        <li class="main list-para"><p>Have the time to share your thoughts and opinions with experts for each topic with experts for each topic.</p></li>
+      </ul>
+    </div>
+    <div id="four">
+      <ul class="list">
+        <li class="main"><img src="./images/workshop.png" alt="" class="program-img"></li>
+        <li class="main"><h4>Workshop</h4></li>
+        <li class="main list-para"><p>Try creating your own work using open source license rather than just watching at it.</p></li>
+      </ul>
+    </div>
+    <div id="five">
+      <ul class="list">
+        <li class="main"><img src="./images/ignite.png" alt="" class="program-img"></li>
+        <li class="main"><h4>CC Ignite</h4></li>
+        <li class="main list-para"><p>Get opportunities to network with CC affiliates around the world, also after the summit you will get it.</p></li>
+      </ul>
+    </div>
+  </div>
+    <div id="join">
+      <span>SEE THE WHOLE PROGRAM SERIES 2023</span>
```

Comment:
- [ ] As suggested by the previous reviewer, the `SEE THE WHOLE PROGRAM SERIES 2023` text is not supposed to have a background color. On the Behance design, it's captured as a line of text with an underline. Please see the screenshot below and rectify this.

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/226114015-3f0445f7-7a10-48c3-942f-a103ad331bf3.png)

design guideline 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/226114071-0103ae9c-a53a-49c6-81b2-5ee36659786f.png)